OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs

```diff
@@ -135,14 +135,7 @@ public ServerBrowserLogic(Widget widget, Action onStart, Action onExit, string d
 				showIncompatibleCheckbox.OnClick = () => { showIncompatible ^= true; RefreshServerList(); };
 			}
 
-			try
-			{
-				Game.LoadWidget(null, "SERVERBROWSER_IRC", panel.Get("IRC_ROOT"), new WidgetArgs());
-			}
-			catch
-			{
-				Log.Write("debug", "Failed to load server browser IRC chrome layout");
-			}
+			Game.LoadWidget(null, "GLOBALCHAT_PANEL", panel.Get("GLOBALCHAT_ROOT"), new WidgetArgs());
```

Comment:
Mods that don't want this can define a dummy `GLOBALCHAT_PANEL` widget to disable it.  The try/catch makes it impossible to debug any errors in the IRC logic.